tlsobs-api/router.go

```diff
@@ -78,6 +78,12 @@ var routes = Routes{
 		"/api/v1/truststore",
 		TruststoreHandler,
 	},
+	Route{
+		"Truststore",
+		"GET",
+		"/api/v1/issuerStatus",
```

Comment:
`Status` or `stats`?
I'd also prefer to avoid camel case in api endpoints, and use lowercase only.